### DISCUSSION TITLE
Drawio raw file conflicts with Adobe analytics

### DISCUSSION THREAD

**Discussion by Sahayavinodh** [squidfunk/mkdocs-material#7502] [Q&A]
I have designed a website using mkdocs material theme which has drawio raw file embedded. After adding adobe analytics to collect the metrics, its conflicting with drawio raw file and drawio raw file is not getting loaded on the website. Can you please your views on this.

**Comment by kamilkrzyskow** [comment]
Hello @Sahayavinodh,
how do you embed the draw.io image? I guess, it looks something like this?:
```html
<div class="mxgraph" style="max-width:100%;border:1px solid transparent;" data-mxgraph="........"></div>
<script type="text/javascript" src="https://viewer.diagrams.net/js/viewer-static.min.js"></script>
```

This works via JavaScript execution, so adding another JavaScript (like the Adobe Analytics) could lead to some sort of conflict with variable names etc.
I recommend checking out the Browser Developer Tools and investigating any errors in the Console:
- https://developer.chrome.com/docs/devtools
- https://developer.chrome.com/docs/devtools/console

Other than that this is a likely not an issue related solely connected to Material for MkDocs, so better move it upstream to Draw.io or the Adobe Analytics support funnels.

**Comment by niclasheinz** [comment]
There is also a [mkdocs-drawio](https://github.com/onixpro/mkdocs-drawio-file) plugin for mkdocs. 

Try this and the drawio file should load correctly.
